CMakeLists.txt

```diff
@@ -1,43 +1,67 @@
-# Minimum version required
-cmake_minimum_required (VERSION 3.2)
-
-# Project name
+cmake_minimum_required (VERSION 3.13)
 project (osqp)
 
+set(CMAKE_VERBOSE_MAKEFILE ON)
```

Comment:
I would turn on verbose only in a debug mode.

Reply:
Done